STM32F4Discovery-Resources/install.sh

```diff
@@ -11,7 +11,7 @@ apt-get -y install build-essential git libsane:i386 ia32-libs-multiarch autoconf
 # Install STLink
 if ! type "st-flash" > /dev/null; then
     echo "*** Installing ST Link"
-    git clone https://github.com/UCSolarCarTeam/stlink.git /opt/stlink --depth 1
```

Comment:
How come this was changed back to the `texane` repo instead of the `UCSolarCarTeam` one?